ui5/browser/controller/Browser.controller.js

```diff
@@ -361,13 +362,61 @@ sap.ui.define(['sap/ui/core/Component',
          this.websocket.Send("BRREQ:" + JSON.stringify(req));
       },
 
+      updateBReadcrumbs: function(jsonString) {
+        let json = JSON.parse(jsonString);
+        let split = json.path.split("/");
+        let oBreadcrumbs = this.getView().byId("breadcrumbs");
+        oBreadcrumbs.removeAllLinks();
+        for (let i=0; i<split.length; i++) {
+          if (i=== split.length-1) {
+            oBreadcrumbs.setCurrentLocationText(split[i]);
+          } else if ( i === 0) {
+            let link = new Link({text: "/"});
+            link.attachPress(this, this.onBreadcrumbsPress);
+            oBreadcrumbs.addLink(link);
+          } else {
+            let link = new Link({text: split[i]});
+            link.attachPress(this, this.onBreadcrumbsPress);
+            oBreadcrumbs.addLink(link);
+          }
+        }
+      },
+
+     onBreadcrumbsPress: function(oEvent, myThis) {
+        let sId = oEvent.getSource().sId;
+        let oBreadcrumbs = oEvent.getSource().getParent();
+        let oLinks = oBreadcrumbs.getLinks();
+        let path = "/";
+        for (let i = 1; i<oLinks.length; i++) {
+          if (oLinks[i].sId === sId ) {
+            path += oLinks[i].getText();
+            break;
+          }
+          path += oLinks[i].getText() + "/";
+        }
+        return myThis.websocket.Send('SWITCHWORKDIR:' + path);
```

Comment:
Maybe something like:
```suggestion
        return myThis.websocket.Send('CHDIR:' + path);
```
would be explicit enough

Reply:
Fixed in : 7a42e78